internals/daemon/export_test.go

```diff
@@ -62,3 +62,16 @@ func FakeSyscallReboot(f func(cmd int) error) (restore func()) {
 		syscallReboot = old
 	}
 }
+
+var (
+	StatusForbidden    = statusForbidden
+	StatusUnauthorized = statusUnauthorized
+)
+
+type (
+	AccessChecker = accessChecker
+	OpenAccess    = openAccess
+	UserAccess    = userAccess
+	RootAccess    = rootAccess
+	Ucrednet      = ucrednet
```

Comment:
Per above, I think we should export all/most of these types to make this usable externally, so the changes in this file will probably be dropped.